flink-formats/flink-protobuf/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
```suggestion
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
# The ASF licenses this file to You under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with
# the License.  You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```
When I ran the test, I got the following error:
```
2021-04-10 23:05:10,066 ERROR  ||  Could not load service provider for factories.   [org.apache.flink.table.factories.FactoryUtil]
java.util.ServiceConfigurationError: org.apache.flink.table.factories.Factory: file:/path/to/target/scala-2.12/classes/META-INF/services/org.apache.flink.table.factories.Factory:1: Illegal provider-class name: /*
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:588)
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:594)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.parseLine(ServiceLoader.java:1142)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.parse(ServiceLoader.java:1169)
```
Perhaps the comments in this file start with `#`.
https://github.com/apache/flink/blob/release-1.12.2/flink-formats/flink-json/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory#L1-L14

Reply:
Thanks for your advice @laughingman7743. I have fixed the issue.